metrics/web/lib/common/presentation/metrics_theme/config/text_field_config.dart

```diff
@@ -6,10 +6,27 @@ class TextFieldConfig {
   static const TextStyle hintStyle = TextStyle(
     color: ColorConfig.inputSecondaryTextColor,
     fontSize: 16.0,
+    height: 1.0,
+  );
+
+  static const TextStyle errorStyle = TextStyle(
+    color: ColorConfig.accentColor,
+    fontSize: 16.0,
+    height: 1.0,
   );
 
   static const border = OutlineInputBorder(
     borderRadius: BorderRadius.all(Radius.circular(4.0)),
     borderSide: BorderSide.none,
   );
+
+  static const focusedBorder = OutlineInputBorder(
+    borderRadius: BorderRadius.all(Radius.circular(4.0)),
+    borderSide: BorderSide(color: ColorConfig.focusedBorderColor),
+  );
```

Comment:
As we discussed in chat - suggest to move such styles to the specific theme to make sure that when we'll work on light theme - dark theme will not be updated.